### ISSUE TITLE
why map instead of returning tuple?

### ISSUE THREAD

**Issue by ladyada** [adafruit/Adafruit_CircuitPython_FXOS8700#3]
hiya writing some code that uses this and its unclear why 

```
        if self._accel_range == ACCEL_RANGE_2G:
            return map(lambda x: x * _ACCEL_MG_LSB_2G * _SENSORS_GRAVITY_STANDARD,
                       accel_raw)
        elif self._accel_range == ACCEL_RANGE_4G:
            return map(lambda x: x * _ACCEL_MG_LSB_4G * _SENSORS_GRAVITY_STANDARD,
                       accel_raw)
        elif self._accel_range == ACCEL_RANGE_8G:
            return map(lambda x: x * _ACCEL_MG_LSB_8G * _SENSORS_GRAVITY_STANDARD,
accel_raw)
```

is preferrable over something like

```
        factor = 0
        if self._accel_range == ACCEL_RANGE_2G:
            factor = _ACCEL_MG_LSB_2G
        elif self._accel_range == ACCEL_RANGE_4G:
            factor = _ACCEL_MG_LSB_4G
        elif self._accel_range == ACCEL_RANGE_8G:
            factor = _ACCEL_MG_LSB_8G
        return [x * factor * _SENSORS_GRAVITY_STANDARD for x in accel_raw]
```
that way you can print the return'd array directly (the map return isnt printable but is iterable)

thx!

**Comment by tannewt** [comment]
map is an older style of that functional approach. It looks like in python3 it returns an iterable object rather than a list as it does in python2. I agree it should be switched to your suggestion using list comprehension.

**Comment by dhalbert** [comment]
Fixed by #4.
